### DISCUSSION TITLE
Poll after command was sent?

### DISCUSSION THREAD

**Discussion by thesebastianf** [zwave-js/zwave-js-ui#2534] [General]
Hey
in the Zwave2MQTT Settings tab I only can find a cyclic poll for legacy devices.
This would really cause a ton of traffic.

I used Openhab before and there was a setting to poll a device channel after a command was issued.
This was really handy. E.g. the Fibaro FGRM 222 shutters do behave not so nice.
When sending  e.g. a Close / 0 to the device it does report a number of states while closing but never the final state. So it will end something like 30. If a poll could be configured to be executed x seconds after a command this would be nice.
Is there a way?

![image](https://user-images.githubusercontent.com/13987575/179855912-fe5f0517-c866-4dbc-afdd-ff09c7ca1d08.png)

**Comment by robertsLando** [comment]
There is no such settings ATM. @AlCalzone Do you think it makes sense to add it?

**Reply by AlCalzone** [reply]
Z-Wave JS already tries to verify the value after setting it. I also have these shutters and no issue with them:
![shutter](https://user-images.githubusercontent.com/17641229/179936266-b092e93d-56b0-443d-84ab-4d8b3ac6777d.gif)
(don't mind the different application, its also Z-Wave JS behind the scenes)

It goes to 0 first (optimistic update), then the device reports an intermediate state and then 0 after stopping. Could it be that yours isn't calibrated correctly?

**Reply by thesebastianf** [reply]
No I have 14 . All calibrated.

Here is what happens:

Lets say they are Closed.
I issue a command to fully Open.
Homeassistent Shows the device now at first as open  but some Seconds after the device is open it sends another value. Like 26 in the Screenshot.

If I click refresh in the zwavejs Admin UI it does get the correct state from the device. Thats why a thought a Poll might be the solution.

But If yours Work fine .... I need to find why or what my Problem is.

![Screenshot_2022-07-21-04-44-54-033_com.android.chrome.jpg](https://user-images.githubusercontent.com/13987575/180206259-5cbe4d33-4a35-45cf-8a35-a6b6781236d5.jpg)

**Reply by AlCalzone** [reply]
Ok I just double checked since you used the level change commands (and I tested with targetValue), but those behave as expected too.

Please make a [driver log](https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/generating-logs?id=driver-logs) of you controlling the device, loglevel `debug` or `silly` and attach it here as a file via drag&drop, so I can see what exactly happens on your end.

**Reply by thesebastianf** [reply]
Will do. Unfortunately can do it only next week since I am not home the next days.

**Comment by thesebastianf** [comment]
[zwavejs2mqtt-store.zip](https://github.com/zwave-js/zwavejs2mqtt/files/9227529/zwavejs2mqtt-store.zip)

For the log I moved NODE 59 from Full Close to Full Open and then Full Close again after a while. All done via UI in Homeassistent. Nothing else was done to the Node but once UP and once DOWN.

After the last Full Close the state jumped to 
![image](https://user-images.githubusercontent.com/13987575/181995092-d6cd6153-8e71-439f-b97e-8c69e362b279.png)
